Wiki.md

```diff
@@ -0,0 +1,117 @@
+## Intro
```

Comment:
Please put newlines after all headers. This header should just be the name of the wiki page, and H1 style

Reply:
https://github.com/uber/AutoDispose/pull/162/commits/edba53cc061fb8f4d7e2dbe2936c7f329f0c2d96